app/views/items/index.html.erb

```diff
@@ -17,7 +17,9 @@
         <%# <span class="closebtnpopup" data-action="click->postal-code#closeBtnPopup">&times;</span> %>
         <label for="postal_code"></label>
         <input class="form-control" type="text" id="postal_code" name="postal_code" value="<%= params[:postal_code] %>" data-postal-code-target="input" placeholder="Enter your zip code...">
-        <button name="button" id="btn" type="submit" class="btn btn-flat">Submit</button>
+        <div class="d-flex justify-content-center">
+          <button name="button" id="btn" type="submit" class="btn btn-flat">Let start !</button>
```

Comment:
For now, we can fix it and put 'Start' or something similar. In the future, we should look into another 'catchy' word.